src/SDKs/CognitiveServices/dataPlane/Vision/Vision/Generated/Face/Models/SimilarFace.cs

```diff
@@ -55,36 +57,32 @@ public SimilarFaceResult(string faceId = default(string), string persistedFaceId
         /// detection call
         /// </summary>
         [JsonProperty(PropertyName = "faceId")]
-        public string FaceId { get; set; }
+        public System.Guid? FaceId { get; set; }
```

Comment:
FaceId? I do not think we have it here.

Reply:
FaceId is [returned](https://westus.dev.cognitive.microsoft.com/docs/services/563879b61984550e40cbbe8d/operations/563879b61984550f30395237) when requested by faceIds.

Reply:
Resolved.